docs/contributing/BuildFromSource.md

```diff
@@ -76,29 +76,20 @@ In most cases, this is because the option _Use previews of the .NET Core SDK_ in
 
 Note, the [Visual Studio Code C# Extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) is required.
 
-Using Visual Studio Code with this repo requires setting environment variables on command line first.
-Use these command to launch VS Code with the right settings.
-
-On Windows (requires PowerShell):
-
-```ps1
-# The extra dot at the beginning is required to 'dot source' this file into the right scope.
-
-. .\activate.ps1
```

Comment:
The `activate` logic seems unnecessary as I didn't notice any difference with/without using the command, but if anyone has further insight feel free to let me know 🙂